include/common/test.h

```diff
@@ -0,0 +1,15 @@
+/*
+ * Copyright © 2017-2018 The Crust Firmware Authors.
+ * SPDX-License-Identifier: (BSD-3-Clause OR GPL-2.0)
+ */
+
+#ifndef TEST_H
```

Comment:
Not your fault (you started writing this before we moved everything), but this should be `COMMON_TEST_H` here and below.

Reply:
done